src/Microsoft.Health.Fhir.Api/Features/Routing/UrlResolver.cs

```diff
@@ -240,9 +240,18 @@ public Uri ResolveOperationResultUrl(string operationName, string id)
             EnsureArg.IsNotNullOrWhiteSpace(operationName, nameof(operationName));
             EnsureArg.IsNotNullOrWhiteSpace(id, nameof(id));
 
-            if (!string.Equals(operationName, OperationsConstants.Export, StringComparison.OrdinalIgnoreCase))
+            string routName = null;
+
+            switch (operationName)
```

Comment:
Corresponding UTs for this change? We have some in `UrlResolverTests.cs`

Reply:
Unit tests added.